### PR TITLE
Add back `bdb` instance into basic usage examples

### PR DESCRIPTION
**Pull request by sohkai**
I do think we still need this bit... see https://github.com/bigchaindb/bigchaindb-driver/pull/251/files#r99296405.

Fixes some extraneous whitespace and tabs as well. Only commit that has non-cosmetic changes is 534961e.

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -8,6 +8,14 @@ First, make sure you're using Python 3,
 you've :doc:`installed the bigchaindb_driver Python package <quickstart>`,
 and you've :doc:`connected to a BigchainDB node or cluster <connect>`.
 
+For the sake of these examples, we'll assume:
+
+.. ipython::
+
+    In [0]: from bigchaindb_driver import BigchainDB
+
+    In [0]: bdb = BigchainDB('http://bdb-server:9984/api/v1')
```
Comment (ttmc) [inline docs/usage.rst]: The whole point of the new `connect.rst` page was to allow for all the diverse possible URL strings that could go inside the first argument of `BigchainDB()`. Why do we need to assume that it's `'http://bdb-server:9984/api/v1'`? I already told them to assume that they're connected to a BigchainDB node or cluster, with a link to the `connect.rst` page.

Maybe the solution is to edit the `connect.rst` page to tell them to use IPython and to put IPython code blocks everywhere on that page?

Comment (sohkai) [inline docs/usage.rst]: Mmm, not sure how the IPython directives work, but I imagine their scope is limited to their current page. Maybe @sbellem knows, but likely, we'll need to put this bit *somewhere* on this page.

Comment (sohkai) [inline docs/usage.rst]: > Why do we need to assume that it's 'http://bdb-server:9984/api/v1'?

No particular reason, aside from that's what it was before, and that's what another example uses later in the document. We could also use the default node, via `BigchainDB()`, so it's less confusing?

Comment (ttmc) [inline docs/usage.rst]: Okay, so maybe comment that `'http://bdb-server:9984/api/v1'` is just one of many possible URLs and they should use _their URL_, as shown on the `connect.rst` page?

**Pull request merged**